app/main/views/marketplace.py

```diff
@@ -88,11 +88,11 @@ def domain(email):
     if brief.get('sellerSelector', '') == 'someSellers':
         seller_domain_list = [domain(x).lower() for x in brief.get('sellerEmailList', [])]
         return current_user.email_address in brief.get('sellerEmailList', []) \
```

Comment:
should this also do a comparison on lower case?

Reply:
Added this for consistency.